src/main/java/it/gov/pagopa/payment/controller/QRCodePaymentController.java

```diff
@@ -29,6 +29,11 @@ AuthPaymentDTO relateUser(@PathVariable("trxCode") String trxCode,
   AuthPaymentDTO authPayment(@PathVariable("trxCode") String trxCode,
       @RequestHeader("x-user-id") String userId);
 
+  @DeleteMapping("/{trxCode}")
+  @ResponseStatus(code = HttpStatus.OK)
+  void unrelateUserPayment(@PathVariable("trxCode") String trxCode,
```

Comment:
keep the name as the "relateUser" method
```suggestion
  void unrelateUser(@PathVariable("trxCode") String trxCode,
```